Build v3.0.5

xylar opened the pull request:
This older version is needed by a downstream package, https://github.com/conda-forge/apache-airflow-providers-fab-feedstock/pull/36.

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

xylar commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824118179) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824118179](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824118179). Examine the logs at this URL for more detail.</sub>

xylar committed:
[3.5.0] Downgrade to v3.0.5

This is needed by a downstream package,
https://github.com/conda-forge/apache-airflow-providers-fab-feedstock/pull/36

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found some lint.

Here's what I've got...


For **recipe/recipe.yaml**:

 * ❌ There are 1 too many lines.  There should be one empty line at the end of the file.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824272966](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824272966). Examine the logs at this URL for more detail.</sub>

xylar commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

xylar commented:
@pmlandwehr and @synapticarbors, first, let me apologize for the noisy messy PR.

Then, I just want to give you a heads up about what I'm doing here.  I need v3.0.5 for https://github.com/conda-forge/apache-airflow-providers-fab-feedstock/pull/36, so my hope is that (once I can get this building right), I can ask you to make a `3.0.5` branch (or whatever you prefer to call it) from `main` as the base branch for this PR to merge to.  That way, we can leave `main` untainted by this update.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824346489) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824346489](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19824346489). Examine the logs at this URL for more detail.</sub>

synapticarbors commented:
No problem. Just ping me when you're ready. Also, if you're core, feel free to just make the branches and merges as needed.

xylar committed:
fix python specs

xylar commented:
@synapticarbors, I'm not core, so I'm afraid I'll need your help.  Thanks very much!

xylar commented:
@conda-forge-admin, please rerender

xylar commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.01.11.54.3

xylar commented:
@synapticarbors, I think things finally look good.  If you could create that base branch for me, I'd be very grateful!

synapticarbors commented:
There is now a `v3.0.5` branch (https://github.com/conda-forge/flask-sqlalchemy-feedstock/tree/v3.0.5). I think I did this correctly -- I based it on the new docs here:
https://github.com/conda-forge/conda-forge.github.io/pull/2661/files#diff-da3d4cac5cc34e4548284fb72dfff0377b49bd367d4c8eb4efe2fe1aa17d5d10

Let me know if you run into any issues.

xylar commented:
@synapticarbors, wonderful, thanks so much.  What you did seems exactly right.

xylar commented:
I believe this is ready to merge whenever you are comfortable doing so.

xylar commented:
Thank you @synapticarbors!
